src/tools.ts

```diff
@@ -200,6 +200,44 @@ async function _callTool(
   return _convertCallToolResult(serverName, toolName, result, client);
 }
 
+export type LoadMcpToolsOptions = {
+  /**
+   * If true, throw an error if a tool fails to load.
+   *
+   * @default true
+   */
+  throwOnLoadError?: boolean;
+
+  /**
+   * If true, the tool name will be prefixed with the server name followed by a double underscore.
+   * This is useful if you want to avoid tool name collisions across servers.
+   *
+   * @default true
+   */
+  prefixToolNameWithServerName?: boolean;
```

Comment:
should this be opt-in? not sure if i want this if i have only one server

Reply:
there was also a discussion on this briefly in py https://github.com/langchain-ai/langchain-mcp-adapters/issues/18

it's probably good to add this as an option!

Reply:
Made this default to opt-in rather than opt-out.

Reply:
So actually, I think this should default to opt-in for direct use of `loadMcpTools` and opt-out for `MultiServerMcpClient`, as the latter case is more likely to benefit from differentiation of tools by server namespace